src/Microsoft.DotNet.Interactive.HttpRequest/Formatting/HttpResponseFormattingExtensions.cs

```diff
@@ -48,6 +75,43 @@ internal static void FormatAsHtml(this HttpResponse? response, FormatContext con
             return;
         }
 
+        PocketView? output = null;
+
+        if (response is HttpResponse fullResponse)
+        {
+            output = fullResponse.FormatAsHtml();
+        }
+        else if (response is PartialHttpResponse partialResponse)
+        {
+            output =
+                PocketViewTags.div[@class: ContainerClass](
```

Comment:
These can be more concise if you add this at the top:

```csharp
using static Microsoft.DotNet.Interactive.Formatting.PocketViewTags;
```

Reply:
Fixed in latest iteration.